server/api/router.go

```diff
@@ -295,6 +295,10 @@ func createRouter(prefix string, svr *server.Server) *mux.Router {
 	registerFunc(apiRouter, "/leader/resign", leaderHandler.ResignLeader, setMethods(http.MethodPost), setAuditBackend(localLog, prometheus))
 	registerFunc(apiRouter, "/leader/transfer/{next_leader}", leaderHandler.TransferLeader, setMethods(http.MethodPost), setAuditBackend(localLog, prometheus))
 
+	msHandler := newMicroServiceHandlerHandler(svr, rd)
+	registerFunc(apiRouter, "/ms/members/{service}", msHandler.GetMembers, setMethods(http.MethodGet), setAuditBackend(prometheus))
+	registerFunc(apiRouter, "/ms/leader/{service}", msHandler.GetLeader, setMethods(http.MethodGet), setAuditBackend(prometheus))
```

Comment:
I have a question, is it better to use primary or leader here？

Reply:
I found we call it as primary in grafana. 
![image](https://github.com/tikv/pd/assets/23399268/3a7b0a58-2843-441b-b36a-3fdbf02a6b3d)
PTAL @rleungx

Reply:
prefer to use primary

Reply:
replaced

Reply:
It seems we don't need this function right now, so let's delete it for now